core/src/main/java/com/scalar/db/storage/dynamo/DynamoAdmin.java

```diff
@@ -1032,6 +1032,46 @@ columnName, getFullTableName(namespace, table)),
         TableMetadata.newBuilder(tableMetadata).addSecondaryIndex(columnName).build());
   }
 
+  private boolean rawIndexExists(String nonPrefixedNamespace, String table, String indexName) {
```

Comment:
```suggestion
  private boolean nativeIndexExists(String nonPrefixedNamespace, String table, String indexName) {
```
I felt `raw` sounds misleading. Is it more proper to call it `native`?

Reply:
Thank you for your comment! It seems that the prefix "Raw" is used to refer to things not managed by ScalarDB metadata, such as `addRawColumnToTable`. Therefore, the name `rawIndexExists` is being used for now.

Reply:
Thanks! OK, let's keep it as is for this PR.

@brfrn169 It's very minor, but I think `raw` sounds misleading , so let's revisit the naming at some point.